components/lib/crash/src/main/java/mozilla/components/lib/crash/prompt/CrashPrompt.kt

```diff
@@ -29,6 +30,13 @@ internal class CrashPrompt(
             return intent
         }
 
+        fun createReportIntent(context: Context, crash: Crash): Intent {
+            val intent = Intent(context, SendCrashReportService::class.java)
+            crash.fillIn(intent)
+
+            return intent
+        }
```

Comment:
nit: I wonder if we should move this somewhere else (maybe to `SendCrashReportService`?) since this is now not related `CrashPrompt`.

Reply:
Yes, makes sense